src/routes/strategies/[strategy]/+layout.svelte

```diff
@@ -17,19 +17,40 @@
 
 <Breadcrumbs labels={breadcrumbs} />
 
-<main class="ds-container">
+<main class="strategy-layout ds-container">
 	<PageHeading>
 		<h1>{summary.name}</h1>
 		<p class="subtitle">{summary.long_description}</p>
 	</PageHeading>
 
 	<StrategyNav strategyId={summary.id} currentPath={$page.url.pathname} />
-
-	<slot />
+	<div class="subpage">
+		<slot />
+	</div>
 </main>
 
 <style lang="postcss">
 	.subtitle {
 		font: var(--f-ui-md-medium);
 	}
+
+	.strategy-layout :global(.page-heading) {
+		@media (--viewport-lg-up) {
+			grid-column: 1/3;
+		}
+	}
+
+	:global(.strategy-nav) {
+		margin-bottom: 2rem;
+	}
```

Comment:
Be sure to nest `:global` styles _under_ a component-specific (non-global) selector… otherwise this style will affect the `.strategy-nav` class _everywhere_. E.g.:

```css
  .strategy-layout :global(.strategy-nav) {
		margin-bottom: 2rem;
  }
```